devices/tuya.js

```diff
@@ -438,6 +438,7 @@ module.exports = [
         model: 'TS004F',
         vendor: 'TuYa',
         description: 'Wireless switch with 4 buttons',
+        whiteLabel: [{vendor: 'Moes', model: 'ESW-0ZAA-EU'}],
```

Comment:
Since this device has a separate definition in the moes.js file, this is not needed.

Reply:
In such case, can you remove it from this pull request, or do I have to do it somehow?